algorithms/utils/action_distributions.py

```diff
@@ -98,6 +103,11 @@ def _kl_inverse(self, other_log_probs):
     def _kl_symmetric(self, other_log_probs):
         return 0.5 * (self._kl(other_log_probs) + self._kl_inverse(other_log_probs))
 
+    def symmetric_kl_with_uniform_prior(self):
+        probs, log_probs = self.probs, self.logits
+        return 0.5 * ((probs * (log_probs - self.log_uniform_prob)).sum(dim=-1)
+                      + (self.uniform_prob * (self.log_uniform_prob - log_probs)).sum(dim=-1))
```

Comment:
I think this is good. Should work.

Reply:
Have you added the same function for Tuple action distributions?

Reply:
Haven't added this, shall add it